### PR TITLE
Service

### PR DESCRIPTION
**Pull request by MarkPikanovsky**

### CODE REVIEW

File: core/src/main/java/com/lms/system/teacher/crm/core/services/impl/LessonServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    @Autowired
+    private LessonDao dao;
+
+    private ModelMapper mapper = new ModelMapper();
+
+    @Override
+    public LessonDto create(LessonDto dto) {
+        validate(dto);
+        Lesson lesson = mapper.map(dto, Lesson.class);
```
Comment (aleksashyn) [inline core/src/main/java/com/lms/system/teacher/crm/core/services/impl/LessonServiceImpl.java]: Are you sure that it's works?

File: pom.xml [hunk 2]
```diff
@@ -321,6 +327,10 @@
                 <excludes>
                   <exclude>**/config/**/*</exclude>
                   <exclude>**/entity/**/*</exclude>
+                  <exclude>**/exceptions/**/*</exclude>
+                  <exclude>**/*Main.*</exclude>
+                  <exclude>**/*TeacherDaoImpl.*</exclude>
```
Comment (aleksashyn) [inline pom.xml]: It's should be cover

File: core/src/main/java/com/lms/system/teacher/crm/core/services/impl/LessonServiceImpl.java (hunk 1 above)
Comment (MarkPikanovsky) [reply core/src/main/java/com/lms/system/teacher/crm/core/services/impl/LessonServiceImpl.java]: Tests says it`s alright

File: core/src/main/java/com/lms/system/teacher/crm/core/dto/LessonDto.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class LessonDto extends AbstractDto {
+
+    private Long id;
+
+    private LessonPortfolioDto portfolio;
```
Comment (aleksashyn) [inline core/src/main/java/com/lms/system/teacher/crm/core/dto/LessonDto.java]: It's should be id

File: core/src/main/java/com/lms/system/teacher/crm/core/dto/LessonPortfolioDto.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import lombok.Data;
+
+@Data
+public class LessonPortfolioDto extends AbstractDto {
+
+    private Long id;
+
+    private Long teacherId;
+
+    private List<LessonDto> lessons;
```
Comment (aleksashyn) [inline core/src/main/java/com/lms/system/teacher/crm/core/dto/LessonPortfolioDto.java]: It's should be ids

Comment (MarkPikanovsky) [reply core/src/main/java/com/lms/system/teacher/crm/core/dto/LessonPortfolioDto.java]: done

File: core/src/main/java/com/lms/system/teacher/crm/core/dto/LessonDto.java (hunk 3 above)
Comment (MarkPikanovsky) [reply core/src/main/java/com/lms/system/teacher/crm/core/dto/LessonDto.java]: done...and seems it still is working

**Pull request merged**